docusaurus/docs/snippets/plugin-e2e-create-ds-plugin.npm.md

```diff
@@ -0,0 +1,3 @@
+```shell
+npm init @grafana/plugin --pluginName='E2E Test datasource' --orgName='sample-org' --pluginDescription='This is a sample datasource.' --pluginType='datasource' --hasBackend --no-hasGithubWorkflows --no-hasGithubLevitateWorkflow
```

Comment:
why `npm init @grafana/plugin` rather than `npx @grafana/create-plugin@latest`? 

When I tried npm init I got the interactive prompts

Reply:
We Mostly to be consistent across the entire article I guess - we refer to npm [here](https://github.com/grafana/plugin-tools/blob/main/packages/create-plugin/README.md#npm), so I would expect this to work. But if it requires interactive prompts, let's use npx. 👍

Reply:
Across the website we use `npx`. I don't think we use `npm init` anywhere in the website documentation so would also vote we stick to `npx` where possible. Likely the reason for the interactive prompts is the syntax. To forward options to commands with `npm` you need to use `--` between the package name and the args.